model/src/main/kotlin/Project.kt

```diff
@@ -85,6 +85,11 @@ data class Project(
      */
     val vcsProcessed: VcsInfo = vcs.normalize(),
 
+    /**
+     * The description of project.
+     */
+    val description: String = "",
```

Comment:
I tend to not assign a default value here (and instead assign the empty string in `Project.EMPTY`), even if that requires more code changes.

But I'm anyway in the context of https://github.com/oss-review-toolkit/ort/issues/9417 preparing a helper function to create projects (with always the correct `projectType`). Maybe we can postpone this PR until then, so that this change only needs to be done in a single place?

Reply:
But we could also do it the opposite way, merge this first and fixup the default argument later.

Reply:
Not using a default value primary consideration IMO should be whether we need backwards compatible deserialization. As you didn't mention this explicitly, I want to double check whether your intention is to break this compatability and if so why?

Reply:
> But we could also do it the opposite way, merge this first and fixup the default argument later.

I prefer to merge this one first, as that other mentioned change also depends on my node refactoring.

Reply:
> that other mentioned change also depends on my node refactoring.

I don't think it does (strictly), as it's a rather small change. But I'm anyway fine with merging this PR first.